packages/plugins/ansiblerole-insights-client/ansiblerole-insights-client.spec

```diff
@@ -0,0 +1,45 @@
+
```

Comment:
Could you remove this empty line?

Reply:
yes

Reply:
(this is old and github doesn't detect this was from another line)